### ISSUE TITLE
feature request : suppport Ask command

### ISSUE THREAD

**Issue by oguh22** [snipsco/Snips-Jeedom-Plugin-Issues#6]
could be nice to support jeedom "ask" command 
Allowing to create interactive scenarios

**Comment by fwehrle** [comment]
Yes. Would be nice to could answer just "yes" or "no" (or more?) when snips ask us something from a jeedom scenario.
